docs/topics/proc_updating-the-deployment-configuration-of-a-booster.adoc

```diff
@@ -8,7 +8,7 @@ For boosters that use the Fabric8 Maven Plugin, the YAML files are located in th
 [IMPORTANT]
 ====
 The deployment configuration files used by the Fabric8 Maven Plugin and Nodeshift do not have to be full OpenShift resource definitions. 
-Both Fabric8 Maven Plugin and Nodeshift can take the provided files and infer some missing information to create a full OpenShift resource definition.
+Both Fabric8 Maven Plugin and Nodeshift can take the deployment configuration files and provide some missing information to create a full OpenShift resource definition.
```

Comment:
"Infer" and "provide" are quite different in meaning. If we don't like "infer", how about just "add" or "fill in"?

Reply:
Fill in is a phrasal verb, and we're not using them unless absoutely necessary. _Add_ works.